step-templates/microsoft-teams-post-a-message.json

```diff
@@ -56,26 +56,36 @@
       "Id": "2d51e4e8-1f81-444f-934f-21a3051e7423",
       "Name": "Timeout",
       "Label": "Timeout in seconds",
-      "HelpText": "The maximum timout in seconds for the request.",
+      "HelpText": "The maximum timout in seconds for each request.",
       "DefaultValue": "60",
       "DisplaySettings": {
         "Octopus.ControlType": "SingleLineText"
       }
     },
     {
-      "Id": "b8835767-e7fe-4ef9-9492-893ca98950f6",

```

Comment:
Changing the ID of a parameter is a breaking change for existing customers' usage of the parameter. The same goes with the name of the parameter too - Octopus will prompt the users, but given this appears to be a change in functionality, I'd prefer we kept it backward compatible.
 
To control the number of retries, I'd suggest adding a new parameter that works in conjunction with this check box. If its enabled, retries can try up to the number set (default 1).